docs/oscar_references.bib

```diff
@@ -1347,6 +1372,41 @@ @Book{Har77
   doi           = {10.1007/978-1-4757-3849-0}
 }
 
+@Article{Has37,
+  author        = {Hasse, H.},
+  title         = {Noch eine Begründung der Theorie der höheren {D}ifferentialquotienten in einem algebraischen
```

Comment:
```suggestion
  title         = {Noch eine Begründung der Theorie der höheren Differentialquotienten in einem algebraischen
```
@joschmitt DocumenterCitations does not like braces in the title. I think you by accident re-added this here

Reply:
Right, thanks! I thought I removed them all...